plugins/it.xsemantics.dsl/src/it/xsemantics/dsl/XsemanticsModelGenerator.xtend

```diff
@@ -0,0 +1,68 @@
+/*******************************************************************************
```

Comment:
This is just a minor modification of the feature Jens added a while ago. So, not a new feature.

Before, an empty line in the copyright header led to a line `" * \n"`, now it leads to `" *\n"` (no trailing space after the asterisk).

(@mmews-n4 please correct me if this is wrong)